invenio_notifications/config.py

```diff
@@ -9,21 +9,38 @@
 
 """Invenio module for notifications support."""
 
-from flask_babelex import gettext as _
-
-from .backends.email import EmailNotificationBackend
+NOTIFICATIONS_BACKENDS = {}
+"""Notification backends.
 
 NOTIFICATIONS_BACKENDS = {
-    EmailNotificationBackend,
-}
-
-NOTIFICATIONS_DEFAULT_SUBJECT = _("New notification from repository")
-
-
-class NotificationConfig:
-    """Config."""
-
-    backends = NOTIFICATIONS_BACKENDS
-
-
-NOTIFICATIONS_CONFIG = NotificationConfig
+    "email": "EmailBackend",
+    "cern": CERNNotificationsBackend,
+    "slack": SlackBackend,
+    }
+"""
+
+NOTIFICATIONS_BUILDERS = {}
+"""Notification builders.
+
+NOTIFICATIONS_BUILDERS = {
+    "community_submission_create": CommunitySubmissionCreate,
+    "community_submission_accept": CommunitySubmissionAccept,
+    "community_submission_reject": CommunitySubmissionReject,
+    "member_invitation_create": CommunityMemberInvitationCreate,
+    "member_invitation_accept": CommunityMemberInvitationAccept,
+    "member_invitation_reject": CommunityMemberInvitationReject,
+    "request_comment_create": RequestCommentCreate,
+    }
```

Comment:
```suggestion
}
```

nit: formatting, check also other docstrings

Reply:
without the indentation, sphinx will raise the following during `./run-tests.sh` (at least locally):
`docstring of invenio_notifications.config.NOTIFICATIONS_BACKENDS:7:Definition list ends without a blank line; unexpected unindent.`


Either indentation or a complete blank line after the last item is needed.
```py
NOTIFICATIONS_BACKENDS = {
    "email": EmailBackend,
    "cern": CERNNotificationsBackend,
    "slack": SlackBackend,
    }
```

or 

```py
NOTIFICATIONS_BACKENDS = {
    "email": EmailBackend,
    "cern": CERNNotificationsBackend,
    "slack": SlackBackend,

}
```

Which one do you prefer?

Reply:
In that case, we need something like the following:

```python
NOTIFICATIONS_BACKENDS = {}
"""Notification backends.

.. code-block:: python

    NOTIFICATIONS_BACKENDS = {
        "email": "EmailBackend",
        "cern": CERNNotificationsBackend,
        "slack": SlackBackend,
    }
"""
```